src/legacy/core_plugins/interpreter/public/functions/clog.ts

```diff
@@ -20,8 +20,8 @@
 export const clog = () => ({
   name: 'clog',
   help: 'Outputs the context to the console',
-  fn: context => {
-    console.log(context); //eslint-disable-line no-console
+  fn: (context: any) => {
```

Comment:
We have an `ExpressionFunction` generic that can be used for all of these

Reply:
Thanks, yes, will use that in last part of #44197, in this PR I would like to do only first part of #44197.